docs/ja/features/subscriptions.md

```
@@ -16,21 +16,21 @@ GeonicDB は WebSocket を介したリアルタイムイベントストリーミ
 - [クライアント実装](#クライアント実装)
 - [ベストプラクティス](#ベストプラクティス)
 - [トラブルシューティング](#トラブルシューティング)
-- [制約事項](#制約事項)
+- [制約](#制約)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)